src/common/options.rs

```diff
@@ -48,6 +48,33 @@ pub struct Options {
     callback_method: Option<CallbackMethod>,
 }
 
+/// Detect Language value
+///
+/// See the [Deepgram Detect Language feature docs][docs] for more info.
+///
+/// [docs]: https://developers.deepgram.com/docs/language-detection
+#[derive(Debug, PartialEq, Eq, Clone, Hash)]
+pub enum DetectLanguage {
+    #[allow(missing_docs)]
+    Enabled(bool),
+
+    #[allow(missing_docs)]
+    Restricted(Vec<Language>)
+}
+
+/// DetectLanguage Impl
+impl DetectLanguage {
+    pub(crate) fn to_string(&self) -> String {
```

Comment:
This isn't quite right. Combined with the other serialization logic, this will lead to `detect_language=en,es`, where the correct syntax for the query string should be `detect_language=en&detect_language=es`.